edge-hub/core/src/Microsoft.Azure.Devices.Edge.Hub.CloudProxy/TokenHelper.cs

```diff
@@ -34,5 +38,164 @@ public static bool IsTokenExpired(string hostName, string token)
         }
 
         public static TimeSpan GetTokenExpiryTimeRemaining(string hostName, string token) => GetTokenExpiry(hostName, token) - DateTime.UtcNow;
+
+        class SharedAccessSignature
```

Comment:
Actually we have SharedAccessSignature in our code already - we can just use it from there - https://github.com/Azure/iotedge/blob/release/1.2/edge-util/src/Microsoft.Azure.Devices.Edge.Util/SharedAccessSignature.cs#L14

Reply:
removed the newly copied one